### PR TITLE
Quilt support

### PR DESCRIPTION
**Pull request by some-du6e**
QFAPI doesnt support 0.90.7 and this fixes it.

### CODE REVIEW

File: plsbuild
Comment (Globox1997) [inline plsbuild:1]: What is this?

**Comment by Globox1997** [comment]
I guess you could just wait until the other mod loader supports it.

File: plsbuild
Comment (some-du6e) [reply plsbuild:1]: So github actions could build. I forgot to delete.

Comment (some-du6e) [reply plsbuild:1]: [db421e7](https://github.com/Globox1997/ExtendedDrawersAddon/pull/1/commits/db421e72901b26e3c0b06ff68453447dacfab8ca) fixes this

**Comment by Globox1997** [comment]
Isnt this fixed with a later fabric api version?

**Pull request closed without merging**